src/foam/nanos/notification/email/EmailTemplateApplyEmailPropertyService.js

```diff
@@ -37,7 +37,7 @@ foam.CLASS({
       try {
         emailMessage = emailTemplateObj.apply(x, group, emailMessage, templateArgs);
       } catch (Exception e) {
-        throw new NoSuchFieldException("@EmailTemplateApplyEmailPropertyService: emailTemplate.apply has failed. emailTemplate = {id:" + templateName + ", group:" + group + "}" + e);
+        logger.error(new NoSuchFieldException("@EmailTemplateApplyEmailPropertyService: emailTemplate.apply has failed. emailTemplate = {id:" + templateName + ", group:" + group + "}" + e));
```

Comment:
As a tiny optimization, we could move the logger declaration here inside the catch block. That way we don't get it out of the context unless we'll need it.